notices/notice-nonce.json

```diff
@@ -1 +1 @@
-1
+2
```

Comment:
I sure wish we could just store signed hashes of these, but some of them need to be signed before the first account is generated.  Unless we secretly generated a vault on first init, and used account[0] to sign them, even store them server-side.

Anyways, that's a fun idea for another time.